gradle.properties

```diff
@@ -9,4 +9,4 @@ springCloudKubernetesVersion=3.1.0
 
 dockertag = pace-local
 
-generatedBufDependencyVersion=00000000000000.c11a58c69ce3
+generatedBufDependencyVersion=00000000000000.c46eb729f8cf
```

Comment:
Think this needs to be `00000000000000.934197b6cd79`

Reply:
have updated already